src/development/vortex/development/networks/models/model.py

```diff
@@ -104,7 +107,19 @@ def available_metrics(self) -> Union[dict, List[str]]:
         pass
```

Comment:
could you fix the available metrics docstring here as well? it is not correctly rendered:

![image](https://user-images.githubusercontent.com/18413229/114500061-8e61f300-9c51-11eb-838b-a3bd8289c0dd.png)

change it to use `.. code-block:: python`, like:
``` rst
example:

.. code-block:: python

    {
        'val_loss': 'min',
        'accuracy': 'max'
    }

or:

.. code-block:: bash

    ['val_loss', 'accuracy']
```

Reply:
and while at it, could you also fix the bulleted points in `predict()` as well, it is not correctly rendered, add indentation and space:

``` rst
"""Predict function, called in inference
includes preprocess, and postprocess.

Expected:
    - input -> image in tensor/numpy with dim (NHWC) [N: batch, H: height, W: width, C: channel]
    - ouput -> list of tensors each member of the list corresponds
      to prediction of the image, or a tensor of dim (NxP) [N:batch, P: prediction]
"""
```

Reply:
updated,

fyi there is indeed rendering issue with sphinx when using newer docutils (0.17), no matter how much indentation you'll get broken list, seems like you have docutils <= 0.16 installed. I need to downgrade docutils to 0.16 before get correct rendering even with indentation

Reply:
ahh yes, maybe that's the problem, cause in my environment it could be rendered correctly